g2p/mappings/langs/kwk/napa_equiv_ubc.csv

```diff
@@ -1,70 +1,70 @@
-p/,p\u0313
-p\u0315,p\u0313
-p\u2019,p\u0313
-t/,t\u0313
-t\u0315,t\u0313
-t\u2019,t\u0313
-k/,k\u0313
-k\u0315,k\u0313
-k\u2019,k\u0313
-c/,c\u0313
-c\u0315,c\u0313
-c\u2019,c\u0313
-ƛ/,ƛ\u0313
-ƛ\u0315,ƛ\u0313
-ƛ\u2019,ƛ\u0313
-k{1}'{2}ʷ{3},k{1}\u0313{2}ʷ{3}
-k{1}]{2}ʷ{3},k{1}\u0313{2}ʷ{3}
-k{1}\u0315{2}ʷ{3},k{1}\u0313{2}ʷ{3}
-k{1}\u2019{2}ʷ{3},k{1}\u0313{2}ʷ{3}
+p/,p̓
+p̕,p̓
+p’,p̓
+t/,t̓
+t̕,t̓
+t’,t̓
+k/,k̓
+k̕,k̓
+k’,k̓
+c/,c̓
+c̕,c̓
+c’,c̓
+ƛ/,ƛ̓
+ƛ̕,ƛ̓
+ƛ’,ƛ̓
```

Comment:
really not a fan of ’ instead of \u2019 since it's a major confusable but I won't object any more